_posts/about algae

```diff
@@ -0,0 +1,33 @@
+
+# About algae
+
+*Type of algae?*
+
+Chlamydomonas reinhardtti
+
+*What is Chlamydomonas reinhardtti?*
+
+Chlamydomonas is a genus of unicellular green algae about 10µm in diameter. It is a common type of algae widely

```

Comment:
Thanks T,

Good start. Can you link to fluid mechanics for example the algae swim around... what does that do to the nearby flow field. What about the effect of turbulence on interactions between the algae? I don't think we need to go super in depth but just enough to create some curiosity, what do you think?

Reply:
Interesting, i will look into it and see what i can get.